wrappers/src/table_cs.cpp

```diff
@@ -234,6 +235,14 @@ REALM_EXPORT void table_set_int64(Table* table_ptr, size_t column_ndx, size_t ro
 REALM_EXPORT void table_set_int64_unique(Table* table_ptr, size_t column_ndx, size_t row_ndx, int64_t value, NativeException::Marshallable& ex)
 {
     return handle_errors(ex, [&]() {
+        if (table_ptr->find_first_int(column_ndx, value) != not_found)
```

Comment:
Formatting: in c++ we only put curly braces on separate lines for functions. (For whatever reason..)